pom.xml

```diff
@@ -715,6 +715,31 @@
                 </plugins>
             </build>
         </profile>
+
+        <!-- using github.com/google/error-prone-javac is required when running on JDK 8 -->
+        <profile>
+            <id>jdk8</id>
+            <activation>
+                <jdk>1.8</jdk>
```

Comment:
This means this profile is activated if the jdk is 1.8? What happens if the jdk is another version?

Reply:
the configuration in this profile with `-J-Xbootclasspath/p:` is only needed on Java 8. errorprone doesn't support < 8. on > 9 this method of overriding the system javac is not allowed.

more details in https://github.com/google/error-prone/blob/gh-pages/docs/installation.md#maven, sorry I forgot to add this link initially.

Reply:
to be more specific, I think all this is needed because "Using the error-prone-javac is required when running on JDK 8"